gulp/tasks/set-env.js

```diff
@@ -8,6 +8,9 @@ const envs = require('../configs/env');
 const cssConfig = require('../configs/css');
 const imagesConfig = require('../configs/images');
 
+/**
+ * Updates scss variable with the environment-dependent image folder path.
+ */
```

Comment:
not only scss, we ill have to provide values to JS (via a module that exports that variable) & to PHP (via $GLOBALS array i guess)